src/common/DataConversionFlowGraph.cpp

```diff
@@ -150,7 +150,7 @@ Result DataConversionFlowGraph::configure(AudioStream *sourceStream, AudioStream
                 mSource = std::make_unique<SourceI32>(sourceChannelCount);
                 break;
             default:
-                LOGE("%s() Unsupported source format = %d", __func__, sourceFormat);
+                LOGE("%s() Unsupported source format = %s", __func__, oboe::convertToText(sourceFormat));
```

Comment:
In this case, we want to know what strange integer value was returned. So the convertToText() will probably not handle it. So better to cast to (int) and print with %d.